parsing.go

```diff
@@ -146,7 +146,13 @@ func ParseReq(txt string) (*Req, error) {
 	r.Level = level
 
 	parts := strings.SplitN(strings.TrimSpace(txt), "\n", 2)
-	r.Title = parts[0]
-	r.Body = formatBodyAsHTML(parts[1])
+
+	if len(parts) == 2{
+		r.Title = parts[0]
+		r.Body = formatBodyAsHTML(parts[1])
+	} else {
+		return nil, fmt.Errorf("Requirement body must not be empty: %s\n", r.ID)
+	}
```

Comment:
Just a detail: We prefer to fail fast - as in when reading the code. "if not ok {return error}" without "else". Otherwise you'd have to move the last return in the "if" block, but then go would complain IIRC.